gems/aws-sdk-core/lib/aws-sdk-core/checksums.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'tempfile'
+require 'digest'
+require_relative 'checksums/digest'
+require_relative 'checksums/digest32'
+require_relative 'checksums/md5'
+require_relative 'checksums/sha1'
+require_relative 'checksums/sha256'
+require_relative 'checksums/crc32'
+require_relative 'checksums/crc32c'
+
+module AWS::SDK::Core
+  # A utility module for calculating checksums.
+  # @api private
+  module Checksums
+    CHUNK_SIZE = 1 * 1024 * 1024 # one MB
+
+    # @param [String] checksum_algorithm - the name of the algorithm
```

Comment:
I think dashes in yard don't render this way. You just need a space after the param name. If it's just a positional param, I don't think you include the name checksum_algorithm, it would just be `@param [String] the name of the algorithm`